README.md

```diff
@@ -72,57 +72,58 @@ The card works with entities from within the **sensor** domain and displays the
+| animate | boolean | `false` | v0.2.0 | Add a reveal animation to the graph.
+| height | number | `150` | v0.0.1 | Set a custom height of the line graph.
+| line_width | number | `5` | v0.0.1 | Set the thickness of the line.
+| line_color | string/list | `var(--accent-color)` | v0.0.1 | Set a custom color for the graph line, provide a list of colors for multiple graph entries.
+| color_thresholds | list |  | v0.2.3 | Set thresholds for dynamic graph colors, see [Line color object](#line-color-object).
 | color_thresholds_transition | string | `smooth` | v0.4.3 | Color threshold transition, `smooth` or `hard`.
-| decimals | integer | optional | v0.0.9 | Specify the exact number of decimals to show for states.
-| hour24 | boolean | false | v0.2.1 | Set to `true` to display times in 24-hour format.
-| font_size | number | 100 | v0.0.3 | Adjust the font size of the state, as percentage of the original size.
-| font_size_header | number | 14 | v0.3.1 | Adjust the font size of the header, size in pixels.
+| decimals | integer |  | v0.0.9 | Specify the exact number of decimals to show for states.
+| hour24 | boolean | `false` | v0.2.1 | Set to `true` to display times in 24-hour format.
+| font_size | number | `100` | v0.0.3 | Adjust the font size of the state, as percentage of the original size.
+| font_size_header | number | `14` | v0.3.1 | Adjust the font size of the header, size in pixels.
 | align_header | string | `default` | v0.2.0 | Set the alignment of the header, `left`, `right`, `center` or `default`.
 | align_icon | string | `right` | v0.2.0 | Set the alignment of the icon, `left`, `right` or `state`.
 | align_state | string | `left` | v0.2.0 | Set the alignment of the current state, `left`, `right` or `center`.
-| lower_bound | number | optional | v0.2.3 | Set a fixed lower bound for the graph Y-axis.
-| upper_bound | number | optional | v0.2.3 | Set a fixed upper bound for the graph Y-axis.
-| lower_bound_secondary | number | optional | v0.5.0 | Set a fixed lower bound for the graph secondary Y-axis.
-| upper_bound_secondary | number | optional | v0.5.0 | Set a fixed upper bound for the graph secondary Y-axis.
+| lower_bound | number |  | v0.2.3 | Set a fixed lower bound for the graph Y-axis.
+| upper_bound | number |  | v0.2.3 | Set a fixed upper bound for the graph Y-axis.
+| lower_bound_secondary | number |  | v0.5.0 | Set a fixed lower bound for the graph secondary Y-axis.
+| upper_bound_secondary | number |  | v0.5.0 | Set a fixed upper bound for the graph secondary Y-axis.
 
 #### Entities object
-Providing options are optional, entities can be listed directly, see example below.
+Entities may be listed directly (as per `sensor.temperature` in the following example), or defined using
+properties of the Entity object detailed in the following table (as per `sensor.pressure` in the following example):
 
 | Name | Type | Default | Description |
 |------|:----:|:-------:|-------------|
```

Comment:
What do you think about removing `Default` since there's none in this table, or keep it and have the option tables consistent?

Reply:
I thought about this too, at this stage I have left the column to keep them consistent. However one of the other tables is missing this column. I am happy with either.